src/Config.hs

```diff
@@ -72,5 +74,9 @@ configT = do
   cfgFetcher <- Toml.dioptional withFetcherT .= cfgFetcher
   pure Config {..}
 
-loadConfig :: MonadIO m => m Config
-loadConfig = Toml.decodeFile configT "config.toml"
+loadConfig :: MonadIO m => String -> m Config
```

Comment:
I think it is better to use [FilePath](https://hackage.haskell.org/package/filepath-1.4.2.2/docs/System-FilePath.html#t:FilePath) type alias